### ISSUE TITLE
Total download size for game updates

### ISSUE THREAD

**Issue by Rafii2198** [Heroic-Games-Launcher/HeroicGamesLauncher#1039]
### Problem description

Whenever you want to download an update for game, there is no feedback about how much you need to download. This is problematic with connections with limited usage because you can't know how internet much you'll use up.
It can count as Accessibility feature.

### Feature description

When pressing update buttons, Heroic will show much GiB needs to be downloaded.

### Alternatives

If that's not possible then at least some information like, when updating the game it shows how much you downloaded already like `[xxxx MiB]`, it could be simply changed to `[xxxx MiB / XXXX GiB]` just to indicate how much you need to download, without using an calculator.

**Comment by Rafii2198** [comment]
All I know that when you update something through Legendary, it will show some info before you can confirm updating, one of these things is `Download size`
For example for an update to the game called Core I have to download 1548.36 MiB

**Comment by Faresgmr** [comment]
It'a been over a year and this is still a thing 
I just want to know so I don't exceed my data cap

**Comment by Cycle9898** [comment]
I'm also interested by this feature because I have a data cap too.
Any news on this ?

**Comment by CommandMC** [comment]
Hello all
The Download Manager UI shows the total size of an installation/update (the "1.46 GiB" in the screenshot):
![image](https://github.com/user-attachments/assets/921892c8-0ad9-4ba7-9eec-4f5334c3eaaa)

I understand some might want to know the size of an update before starting it. However, I'm unsure whether we want to add an additional step to updating a game (considering most people just want to update without worrying about the size). A potential workaround would be pausing the Download Manager right after adding your desired game, or simply quickly navigating to it and deciding whether to let it continue downloading.

I'm going to close this for now, as the original feature request is implemented. If there is sufficient need to have the extra pre-update step, please open a new issue about it.
